src/components/MyAccount/myAccount.js

```diff
@@ -1,32 +1,47 @@
 import React from "react";
 
-export default function MyAccount({name, guardian_picture, guardian_name, guardian_number}) {
+export default function MyAccount({name, guardian_picture, guardian_name, guardian_number, qr_code,user_code}) {
     return (
         <div className="account">
             <div className="account-content">
-                <h1>Witaj, {name}</h1>
-                <h2>Twój opiekun:</h2>
+                <h2>Witaj, {name}</h2>
+                <h3>Twój opiekun:</h3>
                 <img className="guardian-picture" src={guardian_picture} alt={guardian_name}/>
-                <h2>{guardian_name}</h2>
-                <h3>Numer telefonu: {guardian_number}</h3>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-                    Ut enim ad minim veniam,
-                    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-                    Duis aute irure dolor in reprehenderit in voluptate velit esse
-                    cillum dolore eu fugiat nulla pariatur . Excepteur sint occaecat cupidatat non proident,
-                    sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-                <div className="account-menu">
-                        <button className="account-button">Lista moich warsztatów</button>
-                        <button className="account-button">Lista moich drużyn</button>
-                        <button className="account-button">Historia akcji</button>
-                        <button className="account-close_button">Zamknij</button>
+                <h4>Imie i Nazwisko: {guardian_name}</h4>
```

Comment:
Uuuunikałbym stosowanie znaczników `<h1-h6>` do zwykłych nagłówków. Tymi znacznikami można oznaczać takie elementy jak np tytuł karty czy zakładki `<h1>Harmonogram</h1>`
Ewentualnie coś takiego jak wyżej - "Witaj, {name}"
Lepiej to zastąpić zwykłym `<span>` lub `<p>`
Dodatkowo trzeba pamiętać, że znaczniki typu `<h1-h6>, <span>, <p>` mają swoje domyślne stylowania w przeglądarkach. Czasami jest to użyteczne, a czasami nie. Z pewnością nie należy tego stosować do ustawiania wielkości elementów, bo na przykład domyślnie `<h1>` w Chrome może mieć 28px, a w Fire Fox 24px

Reply:
Dobra to pozmieniam i w następnym gicie poleci poprawione